tests/threshold/threshold-config-rate-filter-alert-flow/input.rules

```diff
@@ -0,0 +1 @@
+drop tcp any any -> any any (dsize:0; sid: 1000001;)
```

Comment:
What does this rule mean to do ?

Reply:
it's a generic per packet drop rule that we convert to alert based on thresholding

Reply:
So, we drop the first N packets, then we alert ?

Reply:
This sort of question I try to answer in the readme, for instance :P